helpers/update-site/constants.js

```diff
@@ -0,0 +1,40 @@
+const CONTENT = 'src/content';
+const THEME = 'src/webhint-theme';
+const USER_GUIDE = `${CONTENT}/docs/user-guide`;
+const REPOSITORY = require('../mktemp')();
+
+const dirs = {
+    // About documentation destination directory.
```

Comment:
Nit: Change to `/** */` comment style to get intellisense

Reply:
done!